resources/views/v1/layout/default.twig

```diff
@@ -86,9 +86,27 @@
 
                     {% if not SANDSTORM %}
                         <li class="dropdown user user-menu">
-                        <span style="cursor:default;color:#fff;padding: 15px;display: block;line-height: 20px;">
-                            <span class="hidden-xs">{{ Auth.user.email }}</span>
-                        </span>
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                                <span class="visible-xs">@</span>
+                                <span class="hidden-xs">{{ Auth.user.email }}</span>
+                            </a>
+
+                            <ul class="dropdown-menu" style="width: unset;">
+                                <li class="user-footer">
+                                    <div class="col-xs-6 text-center">
+                                        <a href="{{ route('profile.index') }}" class="btn btn-default btn-flat">
```

Comment:
I'm not a fan of buttons here. They render badly and it's unexpected UI. Can this be a dropdown list?

Reply:
Actually, this is from [the official main header template](https://adminlte.io/docs/2.4/main-header), what is painful to work without the original layout.

Also it broken when I was taking screenshots and replace my email address to a smaller one.

Then, I'm reverting, I'll left in the sidebar menu.